roles/openshift_node/tasks/node_system_container.yml

```diff
@@ -16,14 +16,17 @@
   register: update_result
   changed_when: "'Extracting' in update_result.stdout"
   when:
-  - (openshift.common.version is defined) and (openshift.common.version == openshift_version) and ("node" in result.stdout) | bool
+  - openshift.common.version is defined
```

Comment:
It might make sense to use the default filter and another variable to avoid having to test 'is defined'.

in vars/main.yml:
```
l_openshift_common_version: "{{ openshift.common.version | default('v0') }}"
```